mpu6050_userspace/mpu6050_get_temp_C_and_F_with_w.log

```diff
@@ -0,0 +1,65 @@
+                       Temperature                               
+-----------------------------------------------------------------
+Timestamp: 21:41:53.289321822
+Temp Celsiy 26.742
+Temp Fahrenheit 80.135
+Temp long_word
+Celsius 27.30647058823529411765 
+Fahrenheit 81.15164705882352941177
+-----------------------------------------------------------------
+                       Temperature                               
+-----------------------------------------------------------------
+Timestamp: 21:41:54.438847468
+Temp Celsiy 27.401
+Temp Fahrenheit 81.321
+Temp long_word
+Celsius 27.40058823529411764706 
```

Comment:
What is the value of least significant bit for this sensor? Are 20 digits after comma enough?